### PR TITLE
Add new feature to specify source

### PR DESCRIPTION
**Pull request by hatvik**
Tested on Win 2012 R2.

### CODE REVIEW

**Comment by rvdh** [comment]
Bump. @hatvik: please respond to @ferventcoder - I am currently in need of this feature!

File: lib/puppet/provider/dism/dism.rb [hunk 1]
```diff
@@ -43,6 +43,12 @@ def create
     if resource[:answer]
       cmd << "/Apply-Unattend:#{resource[:answer]}"
     end
+    if resource[:source]
+      cmd << "/Source:#{resource[:source]}"
```
Comment (cyberious) [inline lib/puppet/provider/dism/dism.rb]: Should this be quoted as and when there are spaces in the source path.

**Comment by cyberious** [comment]
@rvdh like the changes but we still need the commits squashed down.  Please rebase and repush

**Comment by ferventcoder** [comment]
Ping @hatvik

**Comment by cyberious** [comment]
Closing in favor of #34 , I have rebased and squashed commits, also added the quotes around source to ensure encapsulating of paths that contain spaces

**Pull request closed without merging**